src/t2iapi/device/service.proto

```diff
@@ -27,6 +27,8 @@ service DeviceService {
     /*
     Switch the device into a specific operating mode.
     This manipulation shall result in a Mds having a specific MdsOperatingMode.
+    The manipulated state shall be persistent until a next manipulation call or until the end of the test case
```

Comment:
How is a t2iapi server supposed to know the scope of "the test case execution" ?

I do agree with all parts other than
> or until the end of the test case execution

though.

Reply:
team decided to remove this part. Done

Reply:
as I see, this is not done

Reply:
@jannast  [NerijusKondrotas]  committed 6 hours ago.